saleor/order/utils.py

```diff
@@ -124,7 +124,11 @@ def cancel_order(order, restock):
 
     payments = order.payments.filter(
         is_active=True,
-        charge_status__in=[ChargeStatus.NOT_CHARGED, ChargeStatus.CHARGED])
+        charge_status__in=[
```

Comment:
Since we're including most of the statuses, maybe it would be more readable if we excluded full refunded ones?